### ISSUE TITLE
Feature Request - Add startup command for command line

### ISSUE THREAD

**Issue by vmario89** [mRemoteNG/mRemoteNG#933]
Hi,

i'd like to be able to start mRemoteNG by command line with an argument to say "start the following connection directly at the startup". 

I am using mRemote with integrated external application Cmder/ConEmu. It would be nice to run this directly when starting mRemote to have a suitable shell in front of the blank panel. Should look like the following:

![grafik](https://user-images.githubusercontent.com/5631071/38312286-a8a50384-3821-11e8-8b8d-61adf2ae3566.png)

How do you think about it?

kind regards, Mario

**Comment by sparerd** [comment]
Makes sense - I like it. It would be nice to be able to do more with mRemoteNG from the command line.

_Dev notes:_ 
This might take a bit to implement since we also need to consider the security impact. App must be fully started, connections file unlocked, then start pending connections. Might be similar to how connections from the previous session reconnect, though we will be reading from the cmd line rather than reading from the connections file. Might be a good idea to elevate the reconnection logic into its own service which can queue up connections to be run at a later time. This would also allow us to save "reconnection groups" and potentially share these connection groups between users.

**Comment by emarshah** [comment]
If possible, can we add the startup commands as well to the connections, for example if i want to open logs for a group of connections, package them together and once opened its tailing the logs instead of writing it every time in multi SSH every time.

**Comment by afarazit** [comment]
Is there an update for this feature?

**Comment by CDRO** [comment]
I'd also be interested in having the possibility to start any existing connection from the CLI, either with started mRemoteNG or from a stopped state.

**Comment by Kvarkas** [comment]
what perhaps will be included into next release - idea to give user more flexibility with settings and atributes.

**Comment by Pamapama** [comment]
Any updates? This feature would be awesome...

**Comment by patrickkasie** [comment]
I second this

**Comment by dpc-SMartin** [comment]
This feature would also be great for PAM solutions which automatically start RDP sessions. If we could start those RDP sessions into mremoteng
